### DISCUSSION TITLE
IconLayer Shift on Zoom

### DISCUSSION THREAD

**Discussion by ssontag55** [visgl/deck.gl#6921] [General]
Has Anyone had this issue?
![icons](https://user-images.githubusercontent.com/563334/169161799-c34083ca-4a7d-4560-9808-94cac6495b1b.gif)

**Comment by ssontag55** [comment]
The Icon size did not match icon mapping size and scale.

**Reply by Pessimistress** [reply]
Hard to say without seeing your code. My guess is your `iconMapping` is incorrect.
